demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -424,6 +426,28 @@ def _is_pack_ignore_file_structure_valid(self):
 
         return False
 
+    @error_codes("PA137")
+    def validate_non_ignorable_error(self):
+        """Check if .pack-ignore structure is parse-able"""
```

Comment:
the name of the function, and this docstring, do not match

Reply:
Done